src/mongo_client.ts

```diff
@@ -254,6 +254,9 @@ export interface MongoClientOptions extends BSONSerializeOptions, SupportedNodeC
   srvPoller?: SrvPoller;
   /** @internal */
   connectionType?: typeof Connection;
+
+  /** @internal */
+  [featureFlag: symbol]: any;
```

Comment:
This is also used for the options argument in the constructor, so that's why the change was needed, the internal will keep it out of public docs, maybe when/if we do a public feature flag we would do something but currently I feel like its overkill to put in that logic.